CardanoSharp.Wallet/TransactionBuilding/TokenBundleBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using CardanoSharp.Wallet.Models.Transactions;
```

Comment:
I'd just like to say this is a wonderful implementation on how to go about creating the builder